### PR TITLE
Hudglasses nerf: Flashproof hud glasses can no longer be crafted from cheap sunglasses.

### LINKED ISSUE #10912: You can craft hudglasses witth cheap sunglasses to make flash protection for yourself
## Reproduction: craft a hudglasses with cheap glasses

alexkar made me do this

### PR DESCRIPTION
**Pull request by TheReddDragon**
# General Documentation

### Intent of your Pull Request

This PR solves and closes #10912 , and is a step to the coder redemption arc

### Why is this change good for the game?

This PR removes a meta strategy of crafting flashproof eyewear from non-flashproof eyewear

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Removes the ability to use cheap sunglasses to craft flash-proof hudglasses. Adds a new item, cheap hudglasses. Same hudglasses, but without flash protection. Cheap security hud sunglasses are not included due to powergaming fears.

### What should players be aware of when it comes to the changes your PR is implementing?
No more free flashproof eye protection

### What general grouping does this PR fall under? 
Game rebalancing

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Cheap HUDGlasses require more cable than their more expensive counterparts due to them requiring more janky wiring 

# Changelog

:cl:  
rscadd: Adds cheap HUDSunglasses
bugfix: Fixes being able to get relatively cheap flash protection
tweak: changes the item path of cheap sunglasses
/:cl:

yes, this has been tested

### CODE REVIEW

**Comment by Jarod1200** [comment]
This hurts the antags

**Comment by TheReddDragon** [comment]
@Jarod1200 good, yogs is mrp not an "MUH ANTAGS CANT WIN" server

**Comment by Jarod1200** [comment]
No antag will RP if sec can just game end them with one click of a flash

**Comment by TheReddDragon** [comment]
listen this issue was stated by a mentor, so raise your concerns with him @Jarod1200
